### DISCUSSION TITLE
Opening projects and batch jobs (on different computers)

### DISCUSSION THREAD

**Discussion by AnnaClo** [OpenChrom/openchrom#469] [Q&A]
Hi, I'm struggling to work with project and batch jobs. It's no problem to create a project and a batch job, in a folder that is different from default. However I don't understand how to open an existing project or batch job. I tried importing but I don't understand the dialogue box

**Comment by eselmeister** [comment]
@AnnaClo Use the "Project Explorer". You can see the location via pop-up menu right mouse click in the settings. Additionally, you can link any folder from your system.

![Screenshot from 2023-08-03 06-46-14](https://github.com/OpenChrom/openchrom/assets/686671/2741dc69-2c0f-4267-b23d-c496f04c7c90)
![Screenshot from 2023-08-03 06-46-45](https://github.com/OpenChrom/openchrom/assets/686671/4abfd429-84a3-41ac-a375-d710077a3d6e)
![Screenshot from 2023-08-03 06-48-28](https://github.com/OpenChrom/openchrom/assets/686671/2eb9afb2-765a-4284-b9dc-578830423850)
![Screenshot from 2023-08-03 06-49-10](https://github.com/OpenChrom/openchrom/assets/686671/6fc30b60-fa32-48f7-8f04-e654947c3cab)

**Comment by AnnaClo** [comment]
Yes, I created a project in a folder in my system (on cloud), but I'm stuck when I try to import that project in OpenChrom (on a different computer that also connects to the same cloud folder : 
![image](https://github.com/OpenChrom/openchrom/assets/140986173/5cb095c3-3466-4aec-b22a-1795e79e4627)
![image](https://github.com/OpenChrom/openchrom/assets/140986173/7bd47d3f-4c60-43c8-93de-e11252675b3d)
![image](https://github.com/OpenChrom/openchrom/assets/140986173/33953aba-4970-4125-b6f1-a8a87428df0f)

I would like also to  load (import?) the batch job that I saved within that project if possible.

**Comment by eselmeister** [comment]
First of all, create a simple project. Than copy & paste your batch file (e.g. BatchJob1.obj) into the new project folder:


![Screenshot from 2023-08-04 06-06-24](https://github.com/OpenChrom/openchrom/assets/686671/1e120ed8-f5bd-42f2-9345-6caca29d0535)
![Screenshot from 2023-08-04 06-06-33](https://github.com/OpenChrom/openchrom/assets/686671/f8fda176-bd8f-48a6-b99e-580dc82f2d9e)
![Screenshot from 2023-08-04 06-07-25](https://github.com/OpenChrom/openchrom/assets/686671/3d2b52dc-f33d-436f-8539-2abd418e846d)
![Screenshot from 2023-08-04 06-08-04](https://github.com/OpenChrom/openchrom/assets/686671/c26b3979-beb9-44a5-81ad-dd5143fcf3a4)
